build-tools/packages/build-cli/src/library/repoPolicyCheck/fluidBuildTasks.ts

```diff
@@ -249,20 +171,28 @@ function findTscScripts(json: Readonly<PackageJson>, project: string): string[]
 	addIfDefined(findScript(json, `tsc --project ${project}`));
 	addIfDefined(findFluidTscScript(json, project));
 	addIfDefined(findTscMultiScript(json, project));
-	return tscScripts.length > 0 ? tscScripts : undefined;
+	if (tscScripts.length === 1) {
+		return tscScripts[0];
+	}
+	if (tscScripts.length === 0) {
+		return undefined;
+	}
+	throw new Error(`${project} used in scripts ${tscScripts.join(", ")}`);
 }
 
 interface EslintConfig {
 	parserOptions?: {
-		project?: string | string[];
+		// https://typescript-eslint.io/packages/parser/#project
+		// eslint-disable-next-line @rushstack/no-new-null
+		project?: string | string[] | boolean | null;
 	};
 }
 /**
  * Get a list of build script names that the eslint depends on, based on .eslintrc file.
- * @remarks eslint does not _depend_ on other build tasks. The projects that it references
- * are configuration guides for the eslint parser, and they associated build tasks
- * are not prerequisites. Consider policy updates that confirm those tasks are
- * present, but do not enforce them as prerequisite actions.
+ * @remarks eslint does not depend on build tasks for the projects it references. The
+ * projects configuration guides for the eslint parser. The packages that those projects
```

Comment:
>  The projects configuration guides for the eslint parser.

I don't follow this. Typo?

Reply:
yep - left "for" from prior wording that should be removed. I'll try a further rewording:
eslint does not depend on build tasks for the projects it references. (The projects' configurations guide eslint typescript parser to use original typescript source.) The packages that those projects depend on must be built. So effectively eslint has the same prerequisites as the build tasks for the projects referenced.